src/cfn_yaml/tags.clj

```diff
@@ -34,6 +34,30 @@
   (encode [data]
     data))
 
+;; https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference-findinmap.html
+(defrecord !FindInMap [mapName topLevelKey secondLevelKey]
+  yaml/YAMLCodec
+  (decode [data keywords]
+    data)
+  (encode [data]
+    data))
+
+;; https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference-getatt.html
+(defrecord !GetAtt [logicalNameOfResource attributeName]
+  yaml/YAMLCodec
+  (decode [data keywords]
+    data)
+  (encode [data]
+    data))
+
+;; https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference-join.html
+(defrecord !Join [delimiter list-of-values]
```

Comment:
Seems that the field names in other records are calmelCased (for similarity to CF), so maybe camelcase here too?